src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -385,7 +417,7 @@ export default [
                 return;
             }
             const editAction = () => {
-                if (_.isUndefined(draftMessage)) {
+                if (!draftMessage) {
```

Comment:
The check will work differently if `draftMessage` is an empty string, is it ok?

Reply:
I tested it, works fine! Thanks for catching it though 😄